scripts/system/create/entityProperties/html/tabs/tabs.css

```diff
@@ -0,0 +1,48 @@
+/*
+//  tabs.css
+//
+//  Created by Alezia Kurdis on 02/27/2020

```

Comment:
Date format should be 27 Feb 2020. (Removes ambiguity of dates such as 1/2/2020 - could be 1 Feb 2020 or 2 Jan 2020, depending on which country you live in.)